etw/rust/tracelogging/src/native.rs

```diff
@@ -444,6 +445,7 @@ extern "system" {
         outer_context: usize,
         reg_handle: &mut u64,
     ) -> u32;
+
```

Comment:
Where did this blank line come from?

Meh. It can be removed later if anybody wants to.